spec/spec_helper.rb

```diff
@@ -1,11 +1,13 @@
 require 'capybara/dsl'
 require 'capybara/poltergeist'
+require 'rspec-benchmark'
 require 'simplecov'
 SimpleCov.start
 require_relative '../wulf_app.rb'
 
 Capybara.default_driver = :poltergeist
-Capybara.app = proc { |env| WulfApp.new.call(env) }
+Capybara.app_host = 'http://localhost:8080'
```

Comment:
App is isolated from RSpec thread, which may hurt code coverage. Should have it set up so that everthing is inline, unless the CI environment is testing performance.